rocket/scripts/run_msascore.py

```diff
@@ -216,17 +231,16 @@ def main():
         )
         llgloss.sfc.atom_b_iso = pseudo_Bs.detach()
         llgloss_rbr.sfc.atom_b_iso = pseudo_Bs.detach()
-
-        # refine_sigmaA:
-        llgloss, llgloss_rbr, Ecalc, Fc = rkrf_utils.update_sigmaA(
-            llgloss=llgloss,
-            llgloss_rbr=llgloss_rbr,
-            aligned_xyz=aligned_xyz,
-            constant_fp_added_HKL=constant_fp_added_HKL,
-            constant_fp_added_asu=constant_fp_added_asu,
-        )
-
-        # Rigid body refinement (RBR) step
+        llgloss.sfc.atom_pos_orth = aligned_xyz
+        llgloss.sfc.savePDB(f"{output_directory_path!s}/{msa_name}_preRBR.pdb")
```

Comment:
any reason you want to save preRBR here? Just for diagnosis or also for production mode?